tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/config/fallback/ConfigPropertyGlobalVsMethodTest.java

```
@@ -0,0 +1,76 @@
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+
+/**
+ * Test class for fallback configuration
+ * @author carlosdlr
+ */
+public class ConfigPropertyGlobalVsMethodTest extends Arquillian {
+
+    @Inject
+    private BeanWithFallback bean;
+
+    @Deployment
+    public static WebArchive deployAnotherApp() {
+        JavaArchive testJar = ShrinkWrap
+            .create(JavaArchive.class, "ftConfigFallback.jar")
+            .addClasses(BeanWithFallback.class)
+            .addAsManifestResource(
+            new StringAsset(
+            "org.eclipse.microprofile.fault.tolerance.tck.config.fallback.BeanWithFallback/serviceA/Fallback/" +
+                "fallbackMethod=fallbackGlobalConfForServiceA"+
+                    "\norg.eclipse.microprofile.fault.tolerance.tck.config.fallback.BeanWithFallback/serviceA/Fallback/" +
+                "applyOn=RuntimeException.class"),
+                                   "microprofile-config.properties")
+            .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+            .as(JavaArchive.class);
+
+        return ShrinkWrap
+            .create(WebArchive.class, "ftConfigFallback.war")
+            .addAsLibrary(testJar);
+    }
+
+    /**
+     * Test that shows how the global(file configuration) has precedence over the annotation conf
+     */
+    @Test
+    void propertyGlobalConfAccessTest() {
+        try {
+            bean.serviceA();
+        }
+        catch (IllegalStateException e) {
+            e.printStackTrace();
+        }
+        Assert.assertEquals(bean.getCalledMethod(), "fallbackGlobalConfForServiceA");
```

Better to use a return value, instead of modifying internal state? Also swallowing the exception is not nice :-)